test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -977,18 +985,478 @@ namespace iroha {
      */
     TEST_F(GetAccountDetailExecutorTest, ValidKeyWriter) {
       addPerms({shared_model::interface::permissions::Role::kGetAllAccDetail});
-      auto query = TestQueryBuilder()
-                       .creatorAccountId(account_id)
-                       .getAccountDetail(account_id2, "key", account_id)
-                       .build();
+      auto query =
+          TestQueryBuilder()
+              .creatorAccountId(account_id)
+              .getAccountDetail(kMaxPageSize, account_id2, "key", account_id)
+              .build();
       auto result = executeQuery(query);
       checkSuccessfulResult<shared_model::interface::AccountDetailResponse>(
           std::move(result), [](const auto &cast_resp) {
             ASSERT_EQ(cast_resp.detail(),
-                      R"({"id@domain" : {"key" : "value"}})");
+                      R"({ "id@domain" : { "key" : "value" } })");
           });
     }
 
+    // --------| GetAccountDetail - pagination tests |------------------>8 -----
+
+    class GetAccountDetailPagedExecutorTest : public QueryExecutorTest {
+     public:
+      // added account details, {writer -> {key -> value}}
+      std::map<std::string, std::map<std::string, std::string>> added_data_;
```

Comment:
```suggestion
      std::map<AccountIdType, std::map<AccountDetailKeyType, AccountDetailValueType>> added_data_;
```
so that comment is not required?

Reply:
this makes the map type easier to understand, but I'd leave the comment anyway for clarity